pandas/tests/io/test_compression.py

```diff
@@ -143,3 +143,26 @@ def test_with_missing_lzma_runtime():
         """
     )
     subprocess.check_output([sys.executable, "-c", code], stderr=subprocess.PIPE)
+
+
+@pytest.mark.parametrize(
+    "obj",
+    [
+        pd.DataFrame(
+            100 * [[0.123456, 0.234567, 0.567567], [12.32112, 123123.2, 321321.2]],
+            columns=["X", "Y", "Z"],
+        ),
+        pd.Series(100 * [0.123456, 0.234567, 0.567567], name="X"),
+    ],
+)
+@pytest.mark.parametrize("method", ["to_pickle", "to_json", "to_csv"])
+def test_gzip_compression_level_path(obj, method):
+    """GH#33398 Ideally this test should be repeated for bz2 as well,
```

Comment:
Is there another test we can do to ensure we pass the keyword args through? If even just that the call works I think OK

Reply:
Sure, we can have a test that simply asserts that the call succeeds without asserting anything about size. I'll add.